package.json

```diff
@@ -25,6 +28,12 @@
     "node-couchdb": "^2.0.0"
```

Comment:
If this package is not in use please remove it.

Reply:
Done